notes/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "notes-client",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "axios": "^0.17.1",
+    "cookie": "^0.3.1",
+    "cookie-jwt": "^1.0.3",
+    "firebase": "^4.10.1",
+    "jwt-decode": "^2.2.0",
+    "react": "^16.2.0",
+    "react-bootstrap": "^0.32.1",
+    "react-cookie": "^2.1.4",
+    "react-dom": "^16.2.0",
+    "react-router-dom": "^4.2.2",
+    "react-scripts": "1.1.0",
```

Comment:
interesting... I had to use `yarn add react-redux` to get
```
    "react-redux": "^5.0.7"
```
in the package.json so I could compile your notes front end